packages/cli/src/commands/versions/bump.ts

```diff
@@ -372,51 +372,64 @@ export function createVersionFinder(options: {
     releaseManifest?.packages.map(p => [p.name, p.version]),
   );
   return async function findTargetVersion(name: string) {
-    const existing = found.get(name);
+    let _name = name;
+    const existing = found.get(_name);
     if (existing) {
       return existing;
     }
 
-    console.log(`Checking for updates of ${name}`);
-    const manifestVersion = releasePackages.get(name);
+    console.log(`Checking for updates of ${_name}`);
+
+    const manifestVersion = releasePackages.get(_name);
     if (manifestVersion) {
       return manifestVersion;
     }
 
-    const info = await packageInfoFetcher(name);
+    // TODO: Properly fix inacurate typing here...
+    const info = (await packageInfoFetcher(_name)) as YarnInfoInspectData & {
+      backstage?: { moved?: string };
+    };
+
+    if (
+      info.backstage?.moved &&
+      info.backstage.moved.startsWith('@backstage-community/')
+    ) {
+      _name = info.backstage.moved;
+    }
+
     const latestVersion = info['dist-tags'].latest;
```

Comment:
Hmm, thinking we'd need to re-fetch if we found a redirect? So that we could potentially follow additional redirects but also get the most recent version of the new package.

Stepping back and looking at the whole thing though, I think this might need a larger refactor outside of this version fetcher?

Reply:
Agree - was not the place for the change.